app/controllers/auctions_controller.rb

```diff
@@ -48,28 +45,16 @@ def previous_winners_archive
   end
 
   def previous_winners
-    collection = AuctionQuery.new.public_index
-    @view_model = AuctionCollection.new(current_user, collection)
-
-    @auctions_json = @view_model.auctions.each { |a| AuctionSerializer.new(a, root: false) }.as_json
-
-    respond_to do |format|
-      format.html
-    end
-  end
-
-  rescue_from 'ActiveRecord::RecordNotFound' do
```

Comment:
I'd have to ask @adelevie as well. There must be a reason? I wonder if this was something to do with not revealing information about unpublished auctions or something else.